docs/tutorial-node-implementation.md

```diff
@@ -0,0 +1,494 @@
+
+[Deployment Artifacts](#deployment-artifacts)
+
+[Tips and tricks](#tips-and-tricks)
+
+
+## Create a new Node Type
+
+This chapter is an example of how to create a new `Node Type`.
+We will recreate the Qiskit `Node Type` from the repository `https://github.com/UST-QuAntiL/tosca-definitions-quantil` .
+
+### 1. Configure the Docker setup and start Winery
+
+Before a `Node Type` can be created, first start a winery instance with the main repository set to where the `Node Type` should be created.
+We will use the `https://github.com/UST-QuAntiL/tosca-definitions-quantil` repository for this tutorial.
+To do this, go to the opentosca-docker folder that you have cloned earlier and add a `docker-compose.override.yml` file with the following content and replace `path-to-folder` with the absolute path to the `opentosca-docker` folder:
+
+```yml
+version: '3'
+services:
+  winery:
+    environment:
+      WINERY_REPOSITORY_PATH: /var/opentosca/repository
+      WINERY_REPOSITORY_URL: https://github.com/UST-QuAntiL/tosca-definitions-quantil
+    volumes:
+      - /path-to-folder/opentosca-docker/repository:/var/opentosca/repository
+```
+
+To start only the winery container, you can use the command `docker-compose up winery`.
+
+:information_source: Reusable `Node Types` that, e.g., can be used by multiple applications, should be placed in a common repository. On the other hand, application-specific `Node Types` should be placed in the application-specific repositories.
+The application-specific repository can then import the common repositories.
+Therefore, the user must create a `repositories.json` file in the repository's root directory.
+In this file, all repositories to be imported are listed with their name, repository URL, and branch:
+```json
+[ {
+  "name" : "Normative Types",
+  "url" : "https://github.com/OpenTOSCA/tosca-normative-types",
+  "branch" : "main"
+},
+{
+  "name" : "Common Types",
+  "url" : "https://github.com/OpenTOSCA/tosca-definitions-common",
+  "branch" : "main"
+},
+{
+  "name" : "QuAntiL Types",
+  "url" : "https://github.com/UST-QuAntiL/tosca-definitions-quantil",
+  "branch" : "main"
+} ] 
+```
+The content of the current repository is in the workspace folder.
+
+
+### 2. Create a new Node Type
+
+Open Winery in your browser (default URL: http://localhost:8080) and click on the `Node Types` tab.
+Click on the `Add new` button on the right.
+
+![Create new node type in winery](./images/new_node_type/new.png)
+
+Enter the following values into the dialog and click on `Add`:
+
+| Field      | Value                                     | Explanation |
+|:-----------|:------------------------------------------|:------------|
+| Name       | CustomQiskit                              |             |
+| Versioning | Enabled                                   | (optional)  |
+| Version    | latest                                    | (optional)  |
+| Namespace  | `https://ust-quantil.github.io/nodetypes` |             |
+
```

Comment:
Please add a section that explains how the namespace works and that you can set the "Template" in the corresponding input. Afterwards, it will be used as the default for all new TOSCA defintions that will be created through out the tutorial.
Then use an example value such as `https://my.company.example.org/tosca/quantum` and explain, that the `/nodetypes`, `nodetypeimplementation`... wil be added automatically.